ddtrace/tracer/option.go

```diff
@@ -228,7 +236,21 @@ func newConfig(opts ...StartOption) *config {
 	c.profilerEndpoints = internal.BoolEnv(traceprof.EndpointEnvVar, true)
 	c.profilerHotspots = internal.BoolEnv(traceprof.CodeHotspotsEnvVar, true)
 	c.enableHostnameDetection = internal.BoolEnv("DD_CLIENT_HOSTNAME_ENABLED", true)
-	c.partialFlushMinSpans = internal.IntEnv("DD_TRACE_PARTIAL_FLUSH_MIN_SPANS", 0)
+	if internal.BoolEnv("DD_TRACE_PARTIAL_FLUSH_ENABLED", false) {
+		c.partialFlushEnabled = true
+		c.partialFlushMinSpans = internal.IntEnv("DD_TRACE_PARTIAL_FLUSH_MIN_SPANS", partialFlushMinSpansDefault)
+		// TODO(partialFlush): add a test for the below two cases
```

Comment:
I'll do this before merging the PR, just added the TODO so I wouldn't forget about it.